### PR TITLE
Increase max_x on repository graph to prevent text cutoff

### LINKED ISSUE #188: Repository size graph cuts off some repository names
See screenshot
Screenshot from 2022-03-02 19-47-13

### PR DESCRIPTION
**Pull request by alyaeanyx**
I believe that this should fix #188 in most cases. Please note that I didn't have time to test this yet because running the webapp requires a full Repology database.

### CODE REVIEW

File: repologyapp/views/graphs_map.py [hunk 1]
```diff
@@ -128,11 +128,17 @@ def point_coords(point: _MapPoint) -> Tuple[float, float]:
...
 
-    with doc.tag('g', ('font-family', 'DejaVu Sans,Verdana,Geneva,sans-serif'), ('font-size', 11), ('text-anchor', 'start')):
-        for point in points:
-            if point.text:
-                x, y = point_coords(point)
-                with doc.tag('text', ('stroke-linecap', 'round'), ('stroke-width', 3), fill='#f0f0f0', stroke='#f0f0f0', x=x + 5, y=y + 3):
+    for point in points:
+        if point.text:
+            x, y = point_coords(point)
+            if x / max_x >= 0.9:
```
Comment (AMDmi3) [inline repologyapp/views/graphs_map.py:134]: The condition is inverted and also you need to use `width` here, as `max_x` is maximal data value (e.g. max # of packages), while you are dealing with pixels here.

File: repologyapp/views/graphs_map.py [hunk 2]
```diff
@@ -128,11 +128,17 @@ def point_coords(point: _MapPoint) -> Tuple[float, float]:
...
+                text_anchor = 'start'
+                x_offset = 5
+            else:
+                text_anchor = 'end'
+                x_offset = -5
+            with doc.tag('g', ('font-family', 'DejaVu Sans,Verdana,Geneva,sans-serif'), ('font-size', 11), ('text-anchor', text_anchor)):
+                with doc.tag('text', ('stroke-linecap', 'round'), ('stroke-width', 3), fill='#f0f0f0', stroke='#f0f0f0', x=x + x_offset, y=y + 3):
                     doc.text(point.text)
                 with doc.tag('text', fill='#000000', x=x + 5, y=y + 3):
```
Comment (AMDmi3) [inline repologyapp/views/graphs_map.py:143]: `x_offset` should be used here too.

**Review by AMDmi3 — CHANGES_REQUESTED** [review]

**Comment by alyaeanyx** [comment]
Sorry, you're absolutely right - I think it should be fine like this now

**Review by AMDmi3 — APPROVED** [review]

**Comment by AMDmi3** [comment]
Excellent work, thank you!

**Pull request merged**